data_pipeline/gmail_data/get_gmail_data.py

```diff
@@ -0,0 +1,40 @@
+from typing import Iterable
+import pandas as pd
+
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+from googleapiclient.discovery import Resource
+
+
+def get_gmail_service_for_user_id(secretFile: str, scopes: str, userId: str) -> Resource:
```

Comment:
minor: would change the parameters here and below to `snake_case`

`pylint` would usually pick that up I think but currently we have the `invalid-name` pylint exclusion configured.